constants/constants.go

```diff
@@ -13,3 +13,6 @@ const (
 	SectionUSPUT  SectionID = 11
 	SectionUSPCT  SectionID = 12
 )
+
+var SectionIDNames = []string{"ID0", "ID1", "tcfeu2", "gpp header", "ID4", "ID5", "uspv1", "uspnat",
+	"uspca", "uspva", "uspco", "usput", "uspct"}
```

Comment:
IMHO: It's a bit odd to see names set for the value 0 which is undefined. There are names for sections 1, 4, and 5 - but they're not supported by this library. Based on how the function is used, it's save to ignore values for which are not parsed.

How do you feel about using a map for supported sections?

```
var sectionNamesByID = map[int]string{
  2: "tcfeu2",
  3: "gpp header",
  ...
}
```

Reply:
Works for me.